docs/articles/storage.md

```diff
@@ -243,8 +243,34 @@ To configure an Amazon Simple Storage Service (s3) bucket for zot, use the `stor
     }
 ```
 
-For descriptions of the configurable attributes for `storageDriver`, see the [s3 storage driver](https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/s3.md) project in GitHub.
-
+The following table lists the attributes of `s3` storage driver:
+
```

Comment:
In the code example above this change, `storage` contains an attribute called `storageDriver` and that attribute contains a `name` attribute, which in this example has a value of `s3`.   Did this schema change?  

When you refer to the "attributes of `s3` storage driver," do you mean "attributes of `storageDriver` when configuring s3 storage"?  If so, should `name` be in the table, or is s3 the only type of storage driver supported?

Reply:
Yes, I meant "attributes of `storageDriver` when configuring s3 storage". According to https://zotregistry.dev/v2.0.1/articles/storage/?h=storage#remote-filesystem, only s3 storage driver is supported.

Reply:
You should change line 246 to clarify what you mean:
"The following table lists the configurable attributes of `storageDriver` when configuring s3 storage."

By the way, is `name` no longer a configurable attributes of `storageDriver`?  If not, it should be removed from the code example, but if it still exists, it should be added to the table.

Reply:
Will do.